src/main/java/edu/ucla/library/services/metadata/MetadataSetter.java

```diff
@@ -365,7 +365,7 @@ private void addMetadata(final String... aRow) throws FileNotFoundException, FfP
             }
         } catch (final IOException details) {
             // We want to distinguish between this IOException and one that comes from reading/writing the CSV file
```

Comment:
Is this comment still true? (Since exceptions thrown from both L334 and L335 are caught here)

Reply:
Removed obsolete comment

Reply:
Is this the correct behavior, though? The throwing of FfProbeException seems like something we'd still want to do (when an IOException is thrown from L334, but not L335). SERV-83 doesn't say anything should be changed about the handling of corrupted media files.

Reply:
The exception provides different error messages for missing/not-found files and for unplayable files--behavior I've seen in action with sample CSVs attached to IIIF-1072 (sales_ephraim_audio.csv references a corrupt md5 file, perlich_2021_03_31_audio.csv lists many nonexistent files). The error message for unplayable files isn't very helpful, but that's a limitation with the ffprobe utility and wrapper Java API.

Reply:
Is it okay that the program now exits with a success status if it encounters corrupted media?

Reply:
It works like buckateer, which processes an entire file even if there are missing or broken image files.

Reply:
Just making sure that's okay since that change in behavior isn't in the acceptance criteria for the ticket.

Reply:
Yes, this is what Lisa wants: the "keep on truckin" approach.

Reply:
I can tweak the ticket criteria to include not failing for unreadable media, if we want that on the record.

Reply:
I think that would be helpful for me since I was absent from this discussion

Reply:
Ticket updated to reflect actual work

Reply:
thanks!